structures/occupancytypes.go

```diff
@@ -257,11 +229,23 @@ func OccupancyTypeMap() map[string]OccupancyTypeStochastic {
 	m["IND6"] = ind6()
 	m["REL1"] = rel1()
 	m["RES1-1SNB"] = res11snb()
+	m["RES1-1SNB_MEDWAVE"] = res11snbMedwave()

```

Comment:
interesting - i wouldnt have done this, but we might be able to add logic to search for the damage function at this level. i can see if i can do that in the damage compute logic.

Reply:
Sorry, I thought I heard to include the occupancy type functions within the map. I thought about whether doing this would impact the logic in the NsiFeatureToStructure method, but it looks to me that once NsiFeatureToStructure assigns the correct occupancy type, then if the WaveHeight hazard, the GetDamage(or Content)FunctionForXXXXWave method takes the name of the correct occupancy type and assigns the correct wave damage curve. I understand that this is messy, and could impact logic elsewhere?

Reply:
Since i dont look for the name Res1-1snb_medwave, it will never be discovered given the current code, it does not have any negative impact to the current logic, but it is holding onto data in memory that probably will never get used (but that is not a big deal really, these data structures are pretty small).

Reply:
however, i could change my code in the future to leverage this feature- so i dont mind hanging onto it since you already did it.